### PR TITLE
See what breaks after adding insecure registry test back.

### PR DESCRIPTION
**Pull request by vaikas**
Signed-off-by: Ville Aikas <vaikas@chainguard.dev>

#### Summary
Follow up comment investigation to here:
https://github.com/sigstore/policy-controller/commit/7e457bb59aaa8d6d1f4ad7ff9d52cab7b1b81db5#commitcomment-85428835

### CODE REVIEW

File: .github/workflows/kind-e2e-cosigned.yaml [hunk 1]
```diff
@@ -112,6 +112,11 @@ jobs:
         kubectl rollout status --timeout 5m --namespace cosign-system deployments/webhook
         kubectl wait deployment -n cosign-system --for condition=Available=True --timeout=90s --all
 
+    - name: Run Insecure Registry Tests
+      run: |
+        go install github.com/google/go-containerregistry/cmd/crane
```
Comment (cpanato) [inline .github/workflows/kind-e2e-cosigned.yaml:117]: maybe we can use `  - uses: imjasonh/setup-crane@v0.2` to setup crane

**Review by cpanato — APPROVED** [review]
non blocking comment

File: .github/workflows/kind-e2e-cosigned.yaml (hunk 1 above)
Comment (hectorj2f) [reply .github/workflows/kind-e2e-cosigned.yaml:117]: +1

**Comment by vaikas** [comment]
It really should be part of the cosign test suite and was removed here in the repository shuffle.
https://github.com/sigstore/cosign/blob/main/test/e2e_test_insecure_registry.sh
I'm closing this one and we can add it there instead.

**Pull request closed without merging**